cloudflare.php

```diff
@@ -13,7 +13,6 @@
     exit;
 }
 
-define('CLOUDFLARE_MIN_WP_VERSION', '3.4');
```

Comment:
It's actually better to have it in cloudflare.php compared to Hooks since cloudflare.php is the first file people check and they can see which version is supported immediately.